docs/source/asr/configs.rst

```diff
@@ -546,6 +547,36 @@ The encoder section includes the details about the RNN-based encoder architectur
 config files and also :ref:`nemo.collections.asr.modules.RNNEncoder <rnn-encoder-api>`.
 
 
+CTC Configurations
+------------------
+
+All CTC-based models also support `InterCTC loss <https://arxiv.org/abs/2102.03216>`_. To use it, you need to specify
+2 parameters as in example below
+
+.. code-block:: yaml
+
+   model:
+      # ...
+      interctc:
+        loss_weights: [0.3]
```

Comment:
Note why list of weights is used

Reply:
There is an example right after this one, which explains why list is used by providing multiple values

Reply:
Ah good enough. Thanks.